lib/kernel-manager.js

```diff
@@ -101,11 +100,7 @@ class KernelManager {
         break;
     }
 
-    if (projectPath != null) {
-      kernelStartDir = projectPath;
-    } else {
-      kernelStartDir = currentPath;
-    }
+    const kernelStartDir = projectPath != null ? projectPath : currentPath;
```

Comment:
@buyology you're right (somehow in mind I thought JS would convert the operands to the same type before evaluating the comparison), but I just checked and:
```
$ node
> "null" != null 
true
```